src/web_app/js/call.js

```diff
@@ -492,6 +491,13 @@ Call.prototype.startSignaling_ = function() {
     } else {
       this.pcClient_.startAsCallee(this.params_.messages);
     }
+    // Setup the remote loopback peerconnection client. Give the first
+    // peerConnection some time to setup.
+    if (this.params_.isLoopback && this.params_.isInitiator) {
+      setTimeout(function() {
+        setupLoopback(this.params_);
+      }.bind(this), 2000);
```

Comment:
Turns out this was not needed anymore, I guess I had race conditions when I started but they have now since been eliminated.